src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -8,8 +8,6 @@
     <EnsureRuntimePackageDependencies>false</EnsureRuntimePackageDependencies>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
 
-    <!-- Ensure a portable PDB is emitted for the project. A PDB is needed for crossgen. -->
-    <DebugType>Portable</DebugType>
     <DebugSymbols>true</DebugSymbols>
```

Comment:
Can `DebugSymbols` be set to true unconditionally in the root same way as `DebugType`?